tests/test_benchmark.py

```diff
@@ -5,30 +5,152 @@
 import time
 
 import numpy as np
+import pytest
 
+from timemachine import constants
 from timemachine.fe.model_utils import apply_hmr
 from timemachine.fe.utils import to_md_units
 from timemachine.ff.handlers import openmm_deserializer
 from timemachine.lib import LangevinIntegrator, MonteCarloBarostat, custom_ops
+from timemachine.lib.potentials import Nonbonded, NonbondedInteractionGroup, NonbondedInteractionGroupInterpolated
 from timemachine.md import builders, minimizer
 from timemachine.md.barostat.utils import get_bond_list, get_group_indices
 from timemachine.testsystems.dhfr import setup_dhfr
+from timemachine.testsystems.relative import setup_hif2a_ligand_pair
 
 
-def recenter(conf, box):
+@pytest.fixture(autouse=True, scope="module")
```

Comment:
I'm not sure I understand why `autouse=True` is needed here?

Reply:
Resolved in [70a31c1](https://github.com/proteneer/timemachine/pull/741/commits/70a31c1d95c6cbeb51e57a6ac0d944e9c06295e5)